packages/rendermime/src/renderers.ts

```diff
@@ -1288,12 +1269,13 @@ namespace Private {
     node: HTMLElement,
     resolver: IRenderMime.IResolver,
     linkHandler: IRenderMime.ILinkHandler | null
-  ): Promise<void> {
-    // Handle anchor elements.
-    const anchors = node.getElementsByTagName('a');
-    for (let i = 0; i < anchors.length; i++) {
-      await handlePathAnchor(anchors[i], resolver, linkHandler);
-    }
+  ): Promise<unknown> {
+    const anchors: HTMLAnchorElement[] = Array.from(
+      node.querySelectorAll('a[data-path]')
```

Comment:
This selector makes sure that only `<a>` nodes that have a `data-path` attribute added to them are processed in `handlePaths`. This was causing errors to render without URLs linked properly.